### PR TITLE
docs,client: improve documentation on access permissions

### PR DESCRIPTION
**Pull request by creachadair**
Updates tailscale/corp#17628

### CODE REVIEW

File: acl/acl.go [hunk 1]
```diff
@@ -17,16 +17,29 @@ import (
...
 const (
-	ActionGet      = Action("get")
-	ActionInfo     = Action("info")
-	ActionPut      = Action("put")
+	// ActionGet ("get" in the API) denotes permission to fetch the contents of a secret.
+	ActionGet = Action("get")
+
+	// ActionInfo ("info" in the API) denotes permission to read the metadata
+	// for a secret, including available and active version numbers, but not the
+	// secret values.
```
Comment (bradfitz) [inline acl/acl.go:25]: Does "get" include this?

File: docs/api.md [hunk 2]
```diff
@@ -12,13 +12,32 @@ The peer capability label is `tailscale.com/cap/secrets`.
...
 - Access permission errors report 403 Forbidden.
 - Requests for unknown values report 404 Not found.
 - All other errors report 500 Internal server error.
 
+
+## Permissions
+
+The service defines named _actions_ that are subject to access control:
+
+- `get`: Denotes permission to fetch the contents of a secret.                                                                             |
```
Comment (bradfitz) [inline docs/api.md:28]: With only "get", can I get version numbers?

File: acl/acl.go (hunk 1 above)
Comment (creachadair) [reply acl/acl.go:25]: It does not.

File: docs/api.md (hunk 2 above)
Comment (creachadair) [reply docs/api.md:28]: You cannot _list_ version numbers, but you can request a specific version.

**Review by bradfitz — APPROVED** [review]
Thanks!

**Pull request merged**